tests/js/spec/utils/profiling/profile/eventedProfile.spec.tsx

```diff
@@ -38,7 +38,7 @@ describe('EventedProfile', () => {
         {type: 'C', at: 4, frame: 0},
       ],
       shared: {
-        frames: [{name: 'f0'}, {name: 'f1'}, {name: 'f2'}],
```

Comment:
Why did this test for `EventedProfile` change when you've only added `SampledProfile`?

Reply:
@Zylphrex it was just unused, I noticed it when I was adding the test for the sampled profile, so I removed it. The stack only contains 2 frames, this 3rd one was there by mistake